open_lm/file_utils.py

```diff
@@ -93,7 +93,7 @@ def get_metadata_file(path):
     out = [json.loads(o) for o in out.decode('utf-8').split('\n')[:-1]]
     return out
 
-def get_shards_for_epoch(num_samples, epoch, path):
+def get_shards_for_chunk(num_samples, chunk, path):
```

Comment:
can we define what a chunk is here with a comment? is it a shard? a group of shards?

Reply:
It's a group of shards.
I'll add a comment clarifying this.

Reply:
Updated with a comment on this.